app/screens/settings/about/about.tsx

```diff
@@ -185,8 +209,18 @@ const About = ({componentId, config, license}: AboutProps) => {
                     license={license}
                 />
                 <Subtitle config={config}/>
-                <SettingSeparator lineStyles={styles.lineStyles}/>
             </View>
+            <View
+                style={{
```

Comment:
Move this style to the makeStylesheetfromtheme

Reply:
That being said, what is the purpose of this screen? Just add some space? Wouldn't you be able to achieve that by adding margins to the elements around?

Reply:
that view is a thin line !

> > @matthewbirtch any input on the button design?
> 
> Thanks for checking on that @enahum. Yes, we need to update the button design. @Sudhanva-Nadiger could you use the small tertiary button with a `content-copy` icon?
> 
> <img alt="" width="400" src="https://user-images.githubusercontent.com/2040554/219112999-ad8f949c-9539-4a4e-8f20-d676a3097cb5.png">

i did as per this reference !

Reply:
I will add comment to those self closing view component so that it is more readable! otherwise its kind of confusing 😄